provider/datagen/src/registry.rs

```diff
@@ -86,8 +84,6 @@ macro_rules! create_datagen_provider {
                 $crate::transform::cldr::ListProvider,
                 $crate::transform::uprops::EnumeratedPropertyCodePointTrieProvider,
                 $crate::transform::uprops::ScriptWithExtensionsPropertyProvider,
-                $crate::transform::uprops::EnumeratedPropertyUnicodeSetDataProvider,
-                // Has to go last as it matches all props/ keys.
```

Comment:
Nit, here and below: Please retain the comment, unless the comment is obsolete?

Reply:
It's obsolete.